package.json

```diff
@@ -1,81 +1,79 @@
 {
-  "name": "@skyux/pages",
-  "version": "5.0.0-beta.3",
-  "description": "SKY UX Pages",
+  "name": "pages-workspace",
+  "version": "0.0.0",
   "scripts": {
-    "build": "skyux build-public-library --fullTemplateTypeCheck",
-    "skyux": "skyux",
-    "start": "skyux serve --hmr",
-    "test": "skyux test --coverage library --logFormat none",
-    "test:visual": "skyux e2e --logFormat none"
+    "ng": "ng",
+    "start": "ng serve",
+    "build": "ng build",
+    "watch": "ng build --watch --configuration development",
+    "test": "ng test",
+    "skyux:generate-lib-resources-module": "ng generate @skyux/i18n:lib-resources-module",
+    "lint": "ng lint",
+    "skyux:generate-documentation": "ng generate @skyux-sdk/documentation-schematics:documentation",
+    "postbuild": "npm run skyux:generate-documentation",
+    "e2e": "ng e2e"
   },
-  "keywords": [],
-  "author": "Blackbaud",
-  "license": "ISC",
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/blackbaud/skyux-pages.git"
-  },
-  "bugs": {
-    "url": "https://github.com/blackbaud/skyux-pages/issues"
-  },
-  "homepage": "https://github.com/blackbaud/skyux-pages#readme",
-  "peerDependencies": {
-    "@angular/common": "^9 || ^12",
-    "@angular/core": "^9 || ^12",
-    "@skyux/i18n": "^5.0.0-beta.0",
-    "@skyux/indicators": "^5.0.0-beta.0",
-    "@skyux/layout": "^5.0.0-beta.0",
+  "private": true,
+  "dependencies": {
+    "@angular/animations": "~12.2.0",
+    "@angular/common": "~12.2.0",
+    "@angular/compiler": "~12.2.0",
+    "@angular/core": "~12.2.0",
+    "@angular/forms": "~12.2.0",
+    "@angular/platform-browser": "~12.2.0",
+    "@angular/platform-browser-dynamic": "~12.2.0",
+    "@angular/router": "~12.2.0",
+    "@blackbaud/auth-client": "^2.48.0",
```

Comment:
Possible to remove most of these `@skyux/*` and `@blackbaud/*` packages, or does your library need them?